MatrixKit/Controllers/MXKViewController.m

```diff
@@ -638,13 +638,9 @@ - (void)onKeyboardWillShow:(NSNotification *)notif
 
     // Detect if an external keyboard is used by checking that
     // the bottom of the provided keyboard frame is outside of the screen
-    BOOL hasExternalKeyboard = NO;
     CGRect keyboard = [self.view convertRect:endRect fromView:self.view.window];
     CGFloat height = self.view.frame.size.height;
-    if ((keyboard.origin.y + keyboard.size.height) > height)
-    {
-        hasExternalKeyboard = YES;
-    }
+    BOOL hasExternalKeyboard = keyboard.size.height <= 80;
```

Comment:
It's a shame we need to check external keyboard like this, thanks Apple for that. Can you just move this value `80` to a constant please?